services/worker-manager/src/data.js

```diff
@@ -123,54 +123,59 @@ WorkerPool.expire = async function(monitor) {
 WorkerPool.prototype.reportError = async function({kind, title, description, extra={}}) {
   const errorId = slugid.v4();
 
-  if (this.emailOnError) {
-    let extraInfo = '';
-    if (Object.keys(extra).length) {
-      extraInfo = `
-It includes the extra information:
-
-\`\`\`
-${yaml.safeDump(extra)}
-\`\`\`
-      `.trim();
-    }
-    await this.notify.email({
-      address: this.owner,
-      subject: `Taskcluster Worker Manager Error: ${title}`,
-      content: `
-Worker Manager has encountered an error while trying to provision the worker pool ${this.workerPoolId}:
-
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-${description}
+  try {
+    if (this.emailOnError) {
+      let extraInfo = '';
+      if (Object.keys(extra).length) {
+        extraInfo = `
+            It includes the extra information:
+            
+            \`\`\`
+            ${yaml.safeDump(extra)}
+            \`\`\`
```

Comment:
It looks like this got indented?

Reply:
oh yeah, I did that, it was hard to read. I am wondering if I can write a tiny function that gets in those variables and returns the string, and just replace the template literal with that function call?

Reply:
Yeah, that would be nice.  @imbstack also wrote an un-indenting function, but I've forgotten what it was called.